release

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env ruby
+require 'fog'
+require 'dotenv'
+require 'json'
+require 'digest'
+require 'git'
+
+Dotenv.load
+
+class FileManager
+  ALL_RELEASES = "releases/all.json"
+  LATEST_RELEASE = "releases/latest.json"
+
+  attr_reader :bucket
+
+  def initialize(bucket)
+    @bucket = bucket
+  end
+
+  def upload!(filename, content)
+    file = bucket.files.new(
+      key: filename,
+      body: content,
+      public: true
+    )
+    file.save
+  end
+
+  def url(filename)
+    get_file(filename).public_url
+  end
+
+  def fetch(filename, default: "")
+    bucket.files.head(filename) ? get_file(filename).body : default
+  end
+
+  def all_releases
+    ALL_RELEASES
+  end
+
+  def latest_release
+    LATEST_RELEASE
+  end
```

Comment:
Sorry, I should've been more specific in that other comment. I was suggesting this:

```ruby
class FileManager
  def all_releases
    fetch(ALL_RELEASES, ...)
  end

  def latest_release
    fetch(LATEST_RELEASE, ...)
  end

  private

  def fetch(...)
    # ...
  end
end

class FeedGenerator
  def entire_feed
    full_feed = JSON.parse(storage_manager.all_releases)
    # ...
  end
end
```